db/migrate/20150813115025_create_container_resource_quota.rb

```diff
@@ -0,0 +1,19 @@
+class CreateContainerResourceQuota < ActiveRecord::Migration
+  def change
+    create_table :container_quotas do |t|
+      t.string     :name
+      t.datetime   :creation_timestamp
+      t.string     :resource_version
+      t.string     :ems_ref
```

Comment:
@moolitayer I can't find where you create the `:ems_ref` is this empty at the moment? Anyway I believe we should drop it because the primary keys are `ems, container_project, name` so no need to generate an `:ems_ref`